eclass/acct-group.eclass

```diff
@@ -32,6 +32,14 @@
 # time (e.g. 'fowners' uses it in src_install), IDEPEND if it must be
 # resolvable at install time (e.g. 'fowners' uses it in pkg_preinst),
 # and RDEPEND in every case.
+#
+# Additionally, users can be assigned to the group by manually specifying
+# them in ACCT_GROUP_<UPPERCASE_GROUPNAME>_USERS_ADD variable in make.conf.
+# This variable is a space-separated list of usernames to be added to the
+# group.  The eclass will take care of adding the group to the users'
+# supplementary groups during their pkg_postinst phase.  This feature is
```

Comment:
Grammar: "during the pkg_postinst phase"

However, I would remove the reference to pkg_postinst entirely: it's an unnecessary implementation detail.